components/EditOutivity/index.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+import { useState } from "react";
+
```

Comment:
Hi Yana. Please have a look at the handout, how to use forms. Here are some hints:
1, Since your component is a 'form', I recommend to name the file like that: ``EditOutivityForm/index.js``.